src/styles/python.scss

```diff
@@ -0,0 +1,54 @@
+@import 'global';
+
+#code {
+  align-items: center;
+  background-color: $code-background;
+  border-radius: $rectangle-corner-radius;
+  display: flex;
+  font-family: $code-font;
+  font-size: $code-font-size;
+  height: 350px;
+  margin-right: 20px;
+  padding-left: 30px;
+  width: 450px;
+}
+
+#w {
```

Comment:
these should probably be classes, not ids! (so `.w`)

see [id vs className](https://www.geeksforgeeks.org/difference-between-an-id-and-class-in-html/#:~:text=Difference%20between%20id%20and%20class,can%20apply%20to%20multiple%20elements.) for more info!

Reply:
can we also have it more descriptive like `white`? because i just saw w o b in the other place and was confused what it was